plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/navigation/class-wc-menu.php

```diff
@@ -100,6 +100,8 @@ public function test_add_plugin_items() {
 	 * Test adding an existing menu ID.
 	 */
 	public function test_add_dupliacte_plugin_items() {
```

Comment:
```suggestion
	public function test_add_duplicate_plugin_items() {
```

Maybe a good opportunity to fix this typo?